packages/clerk-js/src/core/auth/cookies/devBrowser.ts

```diff
@@ -22,7 +22,12 @@ export const createDevBrowserCookie = (cookieSuffix: string): DevBrowserCookieHa
   const devBrowserCookie = createCookieHandler(DEV_BROWSER_JWT_KEY);
   const suffixedDevBrowserCookie = createCookieHandler(getSuffixedCookieName(DEV_BROWSER_JWT_KEY, cookieSuffix));
 
-  const get = () => suffixedDevBrowserCookie.get() || devBrowserCookie.get();
+  const get = (mode?: 'only-suffixed') => {
+    if (mode === 'only-suffixed') {
+      suffixedDevBrowserCookie.get();
```

Comment:
@panteliselef Maybe a typo? 😬 
```suggestion
      return suffixedDevBrowserCookie.get();
```

Reply:
You are correct, i must removed it by mistake, i verified that without `return` it would break. So the change is verified that works.